mathjaxandroid/src/main/java/de/timfreiheit/mathjax/android/MathJaxView.java

```diff
@@ -119,12 +129,16 @@ public void onPageFinished(WebView view, String url) {
      * called when webView is ready with rendering LaTex
      */
     protected void rendered() {
-        handler.postDelayed(new Runnable() {
-            @Override
-            public void run() {
-                mWebView.setVisibility(View.VISIBLE);
-            }
-        }, 100);
+        if (onMathJaxRenderListener != null) {
```

Comment:
the behavior with the invisible webview was made to avoid flickering ui.
why should this be ignored when setting a listener. 
listeners should only listen to events without change the views behavior